system-test/bigtable.ts

```diff
@@ -1414,6 +1421,268 @@ describe('Bigtable', () => {
+        });
+        {
+          await op.promise();
+          // Check the expiry time.
+          await backup.getMetadata();
+          assert.deepStrictEqual(backup.expireDate, sourceExpireTime);
+        }
+        // Create another instance
+        const instance = bigtable.instance(generateId('instance'));
+        const destinationClusterId = generateId('cluster');
+        {
+          // Create production instance with given options
+          const instanceOptions: InstanceOptions = {
+            clusters: [
+              {
+                id: destinationClusterId,
+                nodes: 3,
+                location: 'us-central1-f',
+                storage: 'ssd',
+              },
+            ],
+            labels: {'prod-label': 'prod-label'},
+            type: 'production',
+          };
+          const [, operation] = await instance.create(instanceOptions);
+          await operation.promise();
+        }
+        // Create the copy and test the copied backup
+        await testCopyBackup(
+          backup,
+          {
+            cluster: new Cluster(instance, destinationClusterId),
+            id: generateId('backup'),
+            expireTime: copyExpireTime,
+          },
+          instance
+        );
+        await instance.delete();
+      });
+      it('should create backup of a table and copy it on another cluster of the same instance', async () => {
+        const [backup, op] = await TABLE.createBackup(generateId('backup'), {
+          expireTime: sourceExpireTime,
+        });
+        {
+          await op.promise();
+          // Check the expiry time.
+          await backup.getMetadata();
+          assert.deepStrictEqual(backup.expireDate, sourceExpireTime);
+        }
+        const destinationClusterId = generateId('cluster');
+        {
+          // Create destination cluster with given options
+          const [, operation] = await INSTANCE.cluster(
+            destinationClusterId
+          ).create({
+            location: 'us-central1-b',
+            nodes: 3,
+          });
+          await operation.promise();
+        }
+        // Create the copy and test the copied backup
+        await testCopyBackup(
+          backup,
+          {
+            cluster: new Cluster(INSTANCE, destinationClusterId),
+            id: generateId('backup'),
+            expireTime: copyExpireTime,
+          },
+          INSTANCE
+        );
+      });
+      it('should create backup of a table and copy it on another project', async () => {
+        const [backup, op] = await TABLE.createBackup(generateId('backup'), {
+          expireTime: sourceExpireTime,
+        });
+        {
+          await op.promise();
+          // Check the expiry time.
+          await backup.getMetadata();
+          assert.deepStrictEqual(backup.expireDate, sourceExpireTime);
+        }
+        // Create client, instance, cluster for second project
+        const bigtable = new Bigtable(
```

Comment:
nit - I would rename this to something that signifies it's the second project. Maybe `bigtableSecondaryProject`?

Reply:
or `bigtableDestinationProject`?

Reply:
Done